ts/torch_handler/base_handler.py

```diff
@@ -303,3 +306,21 @@ def _is_explain(self):
                 self.explain = True
                 return True
         return False
+
+    def _is_describe(self):
+        if self.context and self.context.get_request_header(0, "describe"):
+            if self.context.get_request_header(0, "describe") == "True":
+                return True
+        return False
+
+    def describe_handle(self):
+        """Customized describe handler
+
+        Returns:
+            dict : A dictionary response.
+        """
+        output_describe = None
```

Comment:
Should this be an empty `dict` instead? This doesn't seem like a good default implementation, why not just a `pass`

Reply:
good catch.